cli/cmds/contract

```diff
@@ -177,6 +179,8 @@ thaw_contract() {
         # level 3 is the default level of Magic Eden, which offers whitelisting and OTC trading
         thaw_level=3 # https://erc721c.com/docs/integration-guide/creator-token-standards/v3/for-creators/transfer-security
         output=$(cast send $tf_address "setTransferSecurityLevelOfCollection(address,uint8,bool,bool,bool)" $contract_address 3 false false false $password --rpc-url "$RPC_URL" --json)
+    elif [ "$token_standard" == "ERC721" ]; then
+        output=$(cast send $contract_address "setFrozen(bool)" true $password --rpc-url "$RPC_URL" --json)
```

Comment:
Should this be `false`?

Reply:
Also need to update the logic in `deploy_contract` 
`if [ $is_icreatortoken -eq 0 ] || [ "$token_standard" == "ERC1155" ]; then`